### DISCUSSION TITLE
How to load image from byte[]?

### DISCUSSION THREAD

**Discussion by SuperJMN** [SixLabors/ImageSharp#2576] [Q&A]
I have seen overloads that take a stream, but no overload takes a byte[]. Is there any other way to load an image from bytes? I mean, the encoded bytes (eg. png encoded byte[]).

Thanks :)

**Comment by br3aker** [comment]
You can create `MemoryStream` from `byte[]` :)

**Comment by JimBobSquarePants** [comment]
https://docs.sixlabors.com/api/ImageSharp/SixLabors.ImageSharp.Image.html#SixLabors_ImageSharp_Image_Load_ReadOnlySpan_System_Byte__
